react/features/app/components/AbstractApp.js

```diff
@@ -22,7 +24,17 @@ export type Props = {
     /**
      * The URL, if any, with which the app was launched.
      */
-    url: Object | string
+    url: Object | string,
```

Comment:
This is typescript code, not javascript. So shouldn't this file be a .ts file instead of a .js?

Reply:
It is the original code. I tried to rename file from .js to .ts, but it caused issues, eg. some import became not not found in file.